src/hugr/hugrmut.rs

```diff
@@ -21,9 +21,6 @@ pub(crate) trait HugrMut {
     /// Panics if the node is the root node.
     fn remove_op(&mut self, node: Node) -> Result<(), HugrError>;
 
-    /// Remove a node from the graph
-    fn remove_node(&mut self, node: Node) -> Result<(), HugrError>;
```

Comment:
I think `remove_node` makes more sense as the name to keep

Reply:
I was mimicking `add_op`, I guess it makes more sense looking at the parameter name.